crates/matrix-sdk-crypto/src/store/memorystore.rs

```diff
@@ -349,6 +349,11 @@ impl CryptoStore for MemoryStore {
         Ok(())
     }
 
+    async fn remove_custom_value(&self, key: &str) -> Result<()> {
```

Comment:
I don't think we should use a custom value for this. We should persist the token as part of the `save_changes()` call and introduce a getter in the trait.